native/src/components/__tests__/SnackbarContainer.spec.tsx

```diff
@@ -20,8 +20,8 @@ jest.mock('react-redux', () => ({
 
 describe('SnackbarContainer', () => {
   const mockDispatch = jest.fn()
-  const mockUseSelector = mocked(useSelector)
-  const mockUseDispatch = mocked(useDispatch)
+  const mockUseSelector = mocked(useSelector<unknown, ReturnType<typeof useSelector>>)
```

Comment:
Hope you don't get too many merge conflicts but basically you should be able to just accept mine most of the time since these kind of stuff was just deleted